src/js/vre/chapter36/config/form.js

```diff
@@ -162,9 +162,13 @@ const formConfig = {
           uiSchema: {
             veteranFullName: fullNameUI,
             veteranDateOfBirth: currentOrPastDateUI('Date of birth'),
-            veteranSocialSecurityNumber: ssnUI,
+            veteranSocialSecurityNumber: _.assign(ssnUI, {
+              'ui:title': 'Social Security number (must have this or a VA file number)',
+              'ui:required': form => !form.veteranVaFileNumber,
+            }),
             veteranVaFileNumber: {
-              'ui:title': 'VA File Number',
+              'ui:title': 'VA file number (must have this or a Social Security number)',
+              'ui:required': form => !form.veteranSocialSecurityNumber,
               'ui:errorMessages': {
                 pattern: 'Your VA file number must be between 7 to 9 digits'
```

Comment:
Left this error message but it's different from the code that was mentioned [here](https://github.com/department-of-veterans-affairs/vets-website/pull/7179#discussion_r164576333).

Reply:
I think this message is correct.

Reply:
@rianfowler I was actually asking if this was conditionally required, since I asked about that on the previous PR. Just double checking on when what is required, since it seems like this is updated?

Reply:
Stacy [clarified](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/7137#issuecomment-361964661) that at least one is required and it should behave like the pensions form. This doesn't perform exactly like the pensions form- I pulled the error message from the other another form since this is the default error message: 

<img width="510" alt="screen shot 2018-01-31 at 16 17 39" src="https://user-images.githubusercontent.com/4998130/35680844-1933bdca-0721-11e8-8785-f79193a88129.png">

The error message could be tweaked to be more accurate though e.g. "should start with a c, 0, or 1".

Reply:
I should have mentioned this in the comments.